src/Umbraco.Web.BackOffice/Controllers/DictionaryController.cs

```diff
@@ -191,6 +193,40 @@ public ActionResult<DictionaryDisplay> GetById(Udi id)
             return _umbracoMapper.Map<IDictionaryItem, DictionaryDisplay>(dictionary);
         }
 
+        /// <summary>
+        /// Change the sort order for media
+        /// </summary>
+        /// <param name="move"></param>
+        /// <returns></returns>
+        public IActionResult PostMove(DictionaryMove move)
```

Comment:
I think you can use the `MoveOrCopy` class in the parameter instead, see for example:
https://github.com/umbraco/Umbraco-CMS/pull/12089/files

Reply:
You're right! I though the "OrCopy" part was a bit misleading in this context, but I should use existing class instead of creating my own. (Which is basically identical but with less properties) I also found console.log that I had forgotten to remove. But removed it now! :)

Reply:
I think Move and Copy works a bit similar as they need the id of the node it self and the parent node id. Move is basically just changing parent node id and copy creates a node at a specific level using parent node id and a generated node id.